src/main/java/vic/cinema/Main.java

```diff
@@ -18,9 +19,10 @@
 
 public class Main {
     private static Injector injector = Injector.getInstance("vic.cinema");
+    private static final Logger logger = Logger.getLogger(Main.class.getName());
```

Comment:
Try
```suggestion
    private static final Logger logger = Logger.getLogger(Main.class);
```
Should work like this as well)

Reply:
I agree :)

Reply:
Done